src/jvmMain/kotlin/com/sdercolin/vlabeler/ui/string/StringsEnglish.kt

```diff
@@ -474,6 +474,8 @@ fun Strings.en(): String = when (this) {
     PreferencesEditor -> "Editor"
     PreferencesEditorDescription -> "Customize the editor's appearance and behavior."
     PreferencesEditorPlayerCursorColor -> "Player cursor color"
+    PreferencesEditorClickToJumToEntry ->
+        "Jump to the cursor position entry by clicking it on canvas in multiple entry edit mode"
```

Comment:
![QQ_1730898459961](https://github.com/user-attachments/assets/54506fcf-c0bb-440d-911c-0076dae0badd)
The title should be simple and short, like "Click to switch entry"
We can add a description to it.